smart_simulation/cli.py

```diff
@@ -1,15 +1,27 @@
 """Console script for smart_simulation."""
+import importlib
+import logging
 import sys
+
 import click
 
+import daiquiri
+from smart_simulation import consumer
+
+daiquiri.setup(level=logging.INFO)
+logger = daiquiri.getLogger(__name__)
+
+
+def sim_customer(customer, days):
+    return consumer.multi_day(customer, days)
+
 
 @click.command()
-def main(args=None):
-    """Console script for smart_simulation."""
-    click.echo("Replace this message by putting your code into "
-               "smart_simulation.cli.main")
-    click.echo("See click documentation at https://click.palletsprojects.com/")
-    return 0
+@click.argument("customer")
```

Comment:
The convention for CLI's is to put either a `-` or `--` in front of the tools options. So `--customer` and `--days`. This was an interesting read: https://unix.stackexchange.com/questions/21852/single-dashes-for-single-character-options-but-double-dashes-for-words

Also, I think this should be an `option` not an `argument`. Read this and see if it makes sense to switch: https://click.palletsprojects.com/en/7.x/parameters/#differences